index.html

```diff
@@ -894,6 +894,10 @@ <h3>Mandatory fields</h3>
             To provide minimum interoperability, the
             following metadata fields of a <a>Thing</a> MUST
             be contained in a compliant Thing Description:
+	    <em>
+	    FIX: this should have an table assertion per row, and this should be informative
+	    or a parent assertion.
+	    </em>
```

Comment:
Yeah, all rows should be assertions.  I have support for that in my tool, just need to do the markup.  There are lots of these in the document.

Reply:
BTW, when I was fixing this I noted that the assertion now labelled "profile-5.1.2.1-thing-mandatory-fields-1--id" says that the id must be "a globally unique urn of the thing".  So... we took OUT the word globally from the TD spec on purpose to resolve a privacy objection.  Basically the only way to get true global uniqueness is to have a central authority.  So anyway... not a good idea to try and put this back in via profiles, we'll just run into the same issue, again.

Reply:
Also... I'm not sure why security is listed here, it's already mandatory.

Reply:
anyway, fixed.  BTW the -- in the id indicates to my tool that these table assertions are sub-parts of the main assertion with the id before the --, so please don't modify these ids.  If we renumber them let's do in another pass to avoid confusion.  Also, descriptive names would be better than section references (which can easily become stale).